handlers/movie.js

```diff
@@ -80,8 +80,8 @@ async function movie(channel, message, params) {
         return;
       }
 
-      channel.send(format('%s [%s]', replaceRandomWords(movie.Title, params, 1), id));
-      channel.send(replaceRandomWords(movie.Plot, params));
+      channel.send(replaceRandomWords(movie.Title, params, 1));
+      channel.send(replaceRandomWords(movie.Plot, _.capitalize(params)));
```

Comment:
Probably don't need case changed in plot text

Reply:
oops, did it backwards